public/apps/configuration/utils/audit-logging-view-utils.tsx

```diff
@@ -0,0 +1,211 @@
+  return (
+    <>
+      {renderGeneralSettingsForLayer(generalConfiguration)}
+
+      <EuiSpacer />
+
+      {renderGeneralSettingsForAttribute(generalConfiguration)}
+
+      <EuiSpacer />
+
+      {renderGeneralSettingsForExclude(generalConfiguration)}
+
+      <EuiSpacer />
+
+      {renderGeneralSettingsForReport(generalConfiguration)}
+    </>
+  );
+}
+
+function renderGeneralSettingsForLayer(generalConfiguration: any) {
+  return (
+    <>
+      <EuiText>
+        <h3>Layer settings</h3>
+      </EuiText>
+
+      <EuiSpacer />
+
+      <EuiFlexGrid columns={2}>
+        {renderTextFlexItem(
+          'REST layer',
+          generalConfiguration.enable_rest ? 'Enabled' : 'Disabled'
+        )}
+        {renderTextFlexItem(
+          'Transport layer',
+          generalConfiguration.enable_transport ? 'Enabled' : 'Disabled'
+        )}
+        {renderTextFlexItem(
+          'REST disabled categories',
+          generalConfiguration.disabled_rest_categories?.join(', ')
+        )}
+        {renderTextFlexItem(
+          'Transport disabled categories',
+          generalConfiguration.disabled_transport_categories?.join(', ')
+        )}
+      </EuiFlexGrid>
+    </>
+  );
+}
+
+function renderGeneralSettingsForAttribute(generalConfiguration: any) {
+  return (
+    <>
+      <EuiText>
+        <h3>Attribute settings</h3>
+      </EuiText>
+
+      <EuiSpacer />
+
+      <EuiFlexGrid columns={2}>
+        {renderTextFlexItem(
+          'Bulk requests',
+          displayBoolean(generalConfiguration.resolve_bulk_requests)
+        )}
+        {renderTextFlexItem(
+          'Resolve indices',
+          displayBoolean(generalConfiguration.resolve_indices)
+        )}
+        {renderTextFlexItem('Request body', displayBoolean(generalConfiguration.log_request_body))}
+        {renderTextFlexItem(
+          'Sensitive headers',
+          displayBoolean(generalConfiguration.exclude_sensitive_headers)
+        )}
+      </EuiFlexGrid>
+    </>
+  );
+}
+
+function renderGeneralSettingsForExclude(generalConfiguration: any) {
+  return (
+    <>
+      <EuiText>
+        <h3>Exclude settings</h3>
+      </EuiText>
+
+      <EuiSpacer />
+
+      <EuiFlexGrid columns={1}>
+        {renderTextFlexItem('Exclude users', displayArray(generalConfiguration.ignore_users))}
+        {renderTextFlexItem('Exclude requests', displayArray(generalConfiguration.ignore_requests))}
+      </EuiFlexGrid>
+    </>
+  );
+}
+
+function renderGeneralSettingsForReport(generalConfiguration: any) {
+  return (
+    <>
+      <EuiText>
+        <h3>Report settings</h3>
+      </EuiText>
+
+      <EuiSpacer />
+
+      <EuiFlexGrid columns={1}>
+        {renderTextFlexItem('Exclude users', displayArray(generalConfiguration.ignore_users))}
+        {renderTextFlexItem('Exclude requests', displayArray(generalConfiguration.ignore_requests))}
+      </EuiFlexGrid>
+    </>
+  );
+}
+
+export function renderComplianceSettings(complianceSettings: any) {
```

Comment:
Suggest we add the type definition for complianceSettings.

Reply:
Sure. Originally any is used to suppress ts warnings.